superset/db_engine_specs/dremio.py

```diff
@@ -0,0 +1,18 @@
+from superset.db_engine_specs.base import BaseEngineSpec
+import sqlalchemy_dremio.pyodbc
+
+class DremioBaseEngineSpec(BaseEngineSpec):
+
+    engine = "dremio"
+
+    _time_grain_functions = {
+        None: "{col}",
+        "PT1S": "DATE_TRUNC('second', {col})",
+        "PT1M": "DATE_TRUNC('minute', {col})",
+        "PT1H": "DATE_TRUNC('hour', {col})",
+        "P1D": "DATE_TRUNC('day', {col})",
+        "P1W": "DATE_TRUNC('week', {col})",
+        "P1M": "DATE_TRUNC('month', {col})",
+        "P0.25Y": "DATE_TRUNC('quarter', {col})",
+        "P1Y": "DATE_TRUNC('year', {col})",
+    }
```

Comment:
Please add line change to last row.

Reply:
I don't get you. Can you please rephrase?

Reply:
> @narendrans you can add the optional requirements in`setup.py`

Done.

Reply:
It looks like I still need to manually add `import sqlalchemy_dremio` in dremio.py. How does superset automatically import the  one in `extras_require={`?